torch/jit/annotations.py

```diff
@@ -300,6 +300,8 @@ def try_ann_to_type(ann, loc):
     if inspect.isclass(ann) and is_tensor(ann):
         return TensorType.get()
     if is_tuple(ann):
+        if len(ann.__args__) == 1 and ann.__args__[0] == ():
```

Comment:
ubernit: does `if ann.__args__ == [()]:` work?

Reply:
surprisingly not!! Future PEP idea? :(